dockerfiles/init/modules/che-kubernetes-helm/templates/ingress.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: che-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /
+    kubernetes.io/ingress.class: "nginx"
+{{ if .Values.tlsEnabled }}
+    kubernetes.io/tls-acme: "true"
+{{ else }}
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+    ingress.kubernetes.io/proxy-read-timeout: "3600"
```

Comment:
Are you sure that there is no need to update read and connect timeout when TLS is disabled?